.gitignore

```diff
@@ -1,4 +1,8 @@
 *~
+jujugui/static/gui/src/app/assets/javascripts/d3.js
```

Comment:
curious why these additions are necessary because of the more generic ones below.